Makefile

```diff
@@ -3,7 +3,8 @@ TEST_VARS ?= tests/vars.yaml
 TEST_SECRETS ?= tests/secrets.yaml
 TEST_OUTFILE := tests/logs/test_minimal_out_$(shell date +%FT%T%Z).log
 TEST_CONFIG ?= tests/ansible.cfg
+TEST_EXTRA_ARGS ?=
 
 test-minimal:
 	mkdir -p tests/logs
-	ANSIBLE_CONFIG=$(TEST_CONFIG) ansible-playbook -v -i $(TEST_INVENTORY) -e @$(TEST_VARS) -e @$(TEST_SECRETS) tests/playbooks/test_minimal.yaml 2>&1 | tee $(TEST_OUTFILE)
+	ANSIBLE_CONFIG=$(TEST_CONFIG) ansible-playbook -v -i $(TEST_INVENTORY) -e @$(TEST_VARS) -e @$(TEST_SECRETS) $(TEST_EXTRA_ARGS) tests/playbooks/test_minimal.yaml 2>&1 | tee $(TEST_OUTFILE)
```

Comment:
I think you'll need another -e for the new args file and use the @ like the others

-e @$(TEST_EXTRA

Reply:
I'd like the extra args to be able to consume also e.g. `--tags` and not limit it to providing extra variables. So if we want to provide an additional variable file, the `-e @` part should go into the TEXT_EXTRA_ARGS variable itself.